sqlserver/datadog_checks/sqlserver/data/conf.yaml.example

```diff
@@ -348,7 +348,7 @@ instances:
         #
         # samples_per_hour_per_query: 4
 
-        ## @param lookback_window - boolean - optional - default: 2 * collection_interval
+        ## @param lookback_window - boolean - optional - default: 3600
```

Comment:
Do we want to be more cautious about changes the default. The longer the lookback window, the greater the overhead.

Reply:
My thinking is, if you're in a scenario where you need to use this param, you're probably trying to capture infrequent queries. So going up an order of magnitude from minutes to hours made sense to me.

Do we even need to specify a default value? If not, I will just remove, because this param is totally optional.

Reply:
The effective default value, i.e. when the `lookback_window` isn't explicitly defined is `2*collection_interval`. `3600` looks confusing.